gradle/libs.versions.toml

```diff
@@ -20,6 +23,9 @@ junit = "5.10.1"
 jsoup = "1.16.1"
 
 [libraries]
+assertj-core = { module = "org.assertj:assertj-core", version.ref = "assertj" }
+kotlinCompileTesting-core = { module = "dev.zacsweers.kctfork:core", version.ref = "kotlinCompileTesting" }
```

Comment:
Just a quick question related to this dependency. What is the difference between this fork with the [original one](https://github.com/tschuchortdev/kotlin-compile-testing)?